File: PHP/Lab-4/q2.php
<!-- 4. Switch Case – Day of the Week
Objective: Write a program that takes a number (1–7) and prints the corresponding day of the week
using switch case. -->
<?php
    if($_SERVER = "POST"){
        $num = $_POST["in"];

        switch ($num) {
            case '1':
                echo "Sunday";
                break;
            case '2':
                echo "Monday";
                break;
            case '3':
                echo "Tuesday";
                break;
            case '4':
                echo "Wednesday";
                break;
            case '5':
                echo "Thursday";
                break;
            case '6':
                echo "Friday";
                break;
            case '7':
                echo "Saturday";
                break;
            default:
                echo "Not a valid number";
                break;
        }
    }
?>

File: Python/Lab-1/q2.py
# 2. Write a Python program that calculates the factorial of a number using a while loop.

num = int(input("Enter a number to get factorial of: "))
fact = 1
i = 1

while(i <= num):
    fact *= i
    i += 1

print(f"Factorial of {num} is {fact}")